### DISCUSSION TITLE
Unable to build examples for Raspberry Pi Pico target

### DISCUSSION THREAD

**Discussion by qwertymodo** [hathach/tinyusb#1132] [Q&A] [answered]
I've been trying to build the example projects for the Raspberry Pi Pico board target, but it seems that the documentation is missing something, because none of the examples will build.  I was able to successfully build for other targets, such as the example in the documentation of building the cdc_msc device example for the feather_nrf52840_express board target, but when I try to build it for the raspberry_pi_pico board target, here's what I get:

```
qwertymodo@ARTANIS:/mnt/d/Users/qwertymodo/Source/Repos/tinyusb/examples/device/cdc_msc$ make BOARD=raspberry_pi_pico all
cmake -S . -B _build/raspberry_pi_pico -DFAMILY=rp2040 -DBOARD=raspberry_pi_pico -DPICO_BUILD_DOCS=0
CMake Error: The source directory "/mnt/d/Users/qwertymodo/Source/Repos/tinyusb/examples/device/cdc_msc/_build/raspberry_pi_pico" does not exist.
Specify --help for usage, or press the help button on the CMake GUI.
/mnt/d/Users/qwertymodo/Source/Repos/tinyusb/hw/bsp/rp2040/family.mk:9: recipe for target '_build/raspberry_pi_pico' failed
make: *** [_build/raspberry_pi_pico] Error 1
```

I've installed all of the toolchain dependencies listed in the Raspberry Pi Pico's getting started docs, but it seems like there's just something missing here.

**Comment by tinkerbeast** [comment]
Not sure exactly where the problem is from the CMake logs. Maybe you can share your full CMakeLists.txt?

I have also attached my CMakeLists.txt which works for m
[CMakeLists.txt](https://github.com/hathach/tinyusb/files/7315833/CMakeLists.txt)
e.

**Reply by qwertymodo** [reply]
Which directory are you getting that CMakeLists.txt file from?

**Comment by tinkerbeast** [comment]
It's not from any specific directory. I just made that CMakeLists.txt for a standalone project.

**Comment by hathach** [comment]
can you provide your PC setup, your OS, toolchain etc.. Can you compile any of other official examples from rpi i.e pico-examples or pico-sdk ?

**Comment by qwertymodo** [comment, marked as answer]
Looks to be a PEBKAC error.  I nuked my build environment and reinstalled everything using the raspberry pi pico docs and it works now.  I think my mistake was trying to make directly without first creating a build directory for cmake.  In any case, works now...
